### PR TITLE
Task 10

### PR DESCRIPTION
**Pull request by SafaaMagdyAwad**
Give me feedback 😃

### CODE REVIEW

File: Tasks/task10.md
Comment (Nora-Lootfy) [inline Tasks/task10.md:1]: I like your illustration, hoping you have found this research useful for you😊

File: app/Http/Controllers/CarController.php [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 namespace App\Http\Controllers;
 
-use App\common;
+// use App\common;
+use App\Common as AppCommon;
```
Comment (Nora-Lootfy) [inline app/Http/Controllers/CarController.php:6]: thanks for considering my note😊

File: app/Http/Controllers/ProductController.php [hunk 2]
```diff
@@ -40,6 +41,10 @@ public function create()
     {
         return view('add_product');
     }
+    public function about()
```
Comment (Nora-Lootfy) [inline app/Http/Controllers/ProductController.php]: personally, i don't like to put the public pages in the crud controller and i will tell you why

if you open the about page, you will find a section called meet our team, is this section belongs to products?

The way i prefer to work with is that and a send it as text file in the group called organizeFiles

if your website has two sections
Public pages ==> like the template of Fashion
Admin pages ==> like the crud of cars, products ...etc

for controllers
- i separate my controllers in directory called `Admin` for crud controllers or any pages for admin only
- for the public pages, it depends on the pages if there are a lot of pages i make more than one controller each one serve part of public pages & if the number of pages is small, i make one controller e.g. `PublicController` and make function for each page 
      - for example
 ```php
class PublicController {
    public function index() {
        // for index page
    }
    public function about() {
        // for about page
     }
    ...
}
```

**Review by Nora-Lootfy — APPROVED** [review]
I want you to know that you are one of the most hardworking people I’ve ever met. Your dedication and effort are truly impressive. Thank you for all the hard work you put in. It’s incredibly promising!😊

File: app/Http/Controllers/ProductController.php (hunk 2 above)
Comment (SafaaMagdyAwad) [reply app/Http/Controllers/ProductController.php]: Thank you very much, I will remember your instructions

**Comment by SafaaMagdyAwad** [comment]
Thank you eng. Nourhan you helped me the most. You are the one who truly
deserves praise.

**Pull request closed without merging**